imagetest/test_suites/packagevalidation/package_test.go

```diff
@@ -38,41 +61,74 @@ func TestStandardPrograms(t *testing.T) {
 func TestGuestPackages(t *testing.T) {
 	utils.LinuxOnly(t)
 	image, err := utils.GetMetadata(utils.Context(t), "instance", "image")
+
 	if err != nil {
 		t.Fatalf("couldn't determine image from metadata")
 	}
-	cmdPrefix := []string{"rpm", "-q", "--queryformat", "'%{NAME}\n'"}
+
+	// What command to list all packages
+	listPackagesCmd := []string{"rpm", "-qa", "--queryformat", "%{NAME}\n"}
 	if strings.Contains(image, "debian") || strings.Contains(image, "ubuntu") {
-		cmdPrefix = []string{"dpkg-query", "-W", "--showformat", "'${Package}\n'"}
+		listPackagesCmd = []string{"dpkg-query", "-W", "--showformat", "${Package}\n"}
+	}
+
+	// What packages to insure are not installed
+	excludePkgs := []string{}
+
+	// What packages to check for and excude for diffrent images
+	requiredPkgs := []string{"google-guest-agent", "google-osconfig-agent"}
+	if strings.Contains(image, "rhel") {
+		requiredPkgs = append(requiredPkgs, "google-compute-engine", "gce-disk-expand", "google-cloud-sdk")
```

Comment:
Everything except sles should check for `google-cloud-sdk`, right now it's only rhel checking for it.
Same with `gce-disk-expand`, rhel and debian are checking but centos isn't.

Reply:
i know debian gets google-cloud-cli and not google-cloud-sdk, but centos should so i'll rework this part so it checks